pkg/binding/bindingpolicy-resolution.go

```diff
@@ -49,23 +49,25 @@ type bindingPolicyResolution struct {
 	requiresSingletonReportedState bool
 }
 
-// objectData stores the resource version and statuscollectors for an object.
+// objectData stores the UID (why?), resource version, create-only bit,
```

Comment:
The UID is cached for the CombinedStatusResolver's use.

Reply:
Thanks